NSS-Day-Cohort-42/rare-API-media-moguls
# Cb get post userid

#### Changes Made
1. modified fixtures files to account for more than one user
2. removed extra tag import in url.py
3. modified list method in views/post.py to retrieve posts by rareuser

​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-get-post-userid
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
python manage.py run server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
```
5. Test app functionality.
> run Monica's nuke.sh script to delete and then reseed the necessary data for posts, users, and rareusers
> python manage.py runserver
> paste http://localhost:8000/posts?rareuser_id=1 into Postman and run GET to make sure only the posts with rareuser id as 1 are returned
> paste http://localhost:8000/posts?rareuser_id=2 into Postman and run GET to make sure only the posts with rareuser id as 2 are returned

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
